cookbooks/cdo-nginx/recipes/default.rb

```diff
@@ -28,7 +28,7 @@
 
 # Get/create the SSL cert via the `ssl_certificate` cookbook resource, if provided.
 # Otherwise, create a self-signed certificate.
-node.default['ssl_certificate']['service']['compatibility'] = 'modern'
+node.default['ssl_certificate']['service']['compatibility'] = 'intermediate'
```

Comment:
Ah yes, I was going to add a comment on this. CloudFront's CloudFormation template doesn't (yet) support the ability to connect to origin web-servers using TLSv1.1 or TLSv1.2 instead of TLSv1.0 (even though CloudFront itself added support the official CFn template has an API lag issue). The 'modern' compatibility setting disables TLSv1.0 because of security vulnerabilities, so I had to roll it back to 'intermediate' in order for CloudFront to connect to EC2 instances directly. I'm still looking into setting up a workaround for this.